template.yaml

```diff
@@ -87,6 +87,12 @@ Conditions:
   DeployAlarms: !Or
     - Condition: IsNotDevelopment
     - !Equals [!Ref DeployAlarmsInDev, true]
+  EnableCloudFront: !Or
+    - !Equals [ !Ref Environment, dev ]
+    - !Equals [ !Ref Environment, build ]
+    # - !Equals [ !Ref Environment, staging ]
```

Comment:
Can we just remove the commented out code if not needed?

Reply:
Reolved. Removed the commented out code